### PR TITLE
Billing Charges API

### PR DESCRIPTION
**Pull request by DXTimer**
This PR adds OpenAPI definitions for the new Billing Charges API.

There is a single endpoint that has been added:

1. `GET /:account/billing/charges` - (`listCharges`) Lists the billing charges for the account.

Belongs to https://github.com/dnsimple/dnsimple-business/issues/1763

### CODE REVIEW

**Review by AGS4NO — APPROVED** [review]

File: fixtures/v2/api/listCharges/success.http [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Connection: close
+X-RateLimit-Limit: 2400
+X-RateLimit-Remaining: 2397
+X-RateLimit-Reset: 1698143967
+Content-Type: application/json; charset=utf-8
+X-WORK-WITH-US: Love automation? So do we! https://dnsimple.com/jobs
+Cache-Control: no-store, must-revalidate, private, max-age=0
+X-Request-Id: a57a87c8-626a-4361-9fb8-b55ca9be8e5d
+X-Runtime: 0.060526
+Transfer-Encoding: chunked
```
Comment (wilson-dnsimple) [inline fixtures/v2/api/listCharges/success.http:12]: Is this correct? The body does not appear to match the [chunked encoding format](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Transfer-Encoding#chunked). This appears to break the Java tests. Apples to the other fixtures too.

Comment (DXTimer) [reply fixtures/v2/api/listCharges/success.http]: Fixtures have been updated. And indeed we do not use chunked encoding in our live environments.

File: content/v2/billing-charges.markdown [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+| Name       | Type      | Description    |
+| ---------- | --------- | -------------- |
+| `:account` | `integer` | The account ID |
+
+### Filters
+
+| Name          | Description                                 |
+| ------------- | ------------------------------------------- |
+| `:start_date` | Only include results after the given date.  |
```
Comment (weppos) [inline content/v2/billing-charges.markdown:35]: We should provide an example of the format.

Comment (weppos) [reply content/v2/billing-charges.markdown:35]: Actually it's in the examples below, so it may be good. Although I would still add a format info.

**Review by weppos — APPROVED** [review]

File: content/v2/billing-charges.markdown (hunk 2 above)
Comment (DXTimer) [reply content/v2/billing-charges.markdown]: Format information added in https://github.com/dnsimple/dnsimple-developer/pull/536/commits/1234b288936a4c74816ccfb65c360e7e65a3b32f

**Pull request merged**